renovate.json5

```diff
@@ -42,9 +42,34 @@
   "packageRules": [
     {
       "description": "Ignore spring-boot-parent updates in EAI template because Camel version specifies compatible Spring version",
+      "matchDatasources": ["maven"],
       "matchPackageNames": ["org.springframework.boot:spring-boot-starter-parent"],
       "matchFileNames": ["refarch-eai/pom.xml"],
       "enabled": false
```

Comment:
Why is this disabled?

Reply:
Because we don't want Spring updates for the EAI template. Camel strictly defines the supported Spring version and thus when we update Camel, Spring Boot version has to be aligned manually.

Reply:
I misunderstood the meaning of `enabled` in this case.